tests/src/contract_data_warning.test.js

```diff
@@ -0,0 +1,38 @@
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
+import { expect } from "../jest";
+import { TransportStatusError } from "@ledgerhq/errors";
+import { waitForAppScreen, zemu } from './test.fixture';
+import Zemu from '@zondax/zemu';
+
+test('[Nano S] Try to blind sign with setting disabled', zemu("nanos", async (sim, eth) => {
+  // disable blind signing
+  await sim.navigateAndCompareSnapshots('.', 'nanos_disable_blind_signing', [2, 0, 0, 3, 0]);
+
+  // we can't use eth.signTransaction because it detects that contract data is disabled and fails early
```

Comment:
Interesting. Thanks for pointing this out in the comments! :)

Reply:
:pet-myself: